set.mm

```diff
@@ -476838,8 +476915,29 @@ The sumset operation can be used for both group (additive) operations and
     ${
       $d .X. a $.  $d .X. e i $.  $d B e i $.  $d E a e i $.  $d G e i $.
       $d I a e i $.  $d R e i $.  $d a ph $.  $d e i ph $.
-      ringlsmss2.1 $e |- ( ph -> E C_ B ) $.
-      ringlsmss2.2 $e |- ( ph -> I e. ( LIdeal ` R ) ) $.
+      ringlsmss1.1 $e |- ( ph -> R e. CRing ) $.
+      ringlsmss1.2 $e |- ( ph -> E C_ B ) $.
+      ringlsmss1.3 $e |- ( ph -> I e. ( LIdeal ` R ) ) $.
+      $( The product to the left with an ideal of a commutative ring is a
```

Comment:
I think the phrase "The product to the left" is a bit confusing. Maybe say something like "The product of an ideal ` I ` of a commutative ring with some set ` E ` is a subset of the ideal." ?

In general, I find the theorem a bit weird, in a commutative ring left ideals are two-sided ideals, see [crng2idl](https://us.metamath.org/mpeuni/crng2idl.html). Perhaps you need this result for something and it was easier to prove it that way?

Reply:
Indeed, the proof would have been shorter using ~crng2idl!
I'll update the comment. I wanted to keep short proofs and this seemed like an interesting closure-like theorem.